src/main/src/jetty/database/ncdfgenerator/changelog.xml

```diff
@@ -0,0 +1,83 @@
+<?xml version="1.0" encoding="UTF-8"?>
+
+<databaseChangeLog
+        xmlns="http://www.liquibase.org/xml/ns/dbchangelog"
+        xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+        xmlns:ext="http://www.liquibase.org/xml/ns/dbchangelog-ext"
+        xsi:schemaLocation="http://www.liquibase.org/xml/ns/dbchangelog http://www.liquibase.org/xml/ns/dbchangelog/dbchangelog-3.0.xsd
+        http://www.liquibase.org/xml/ns/dbchangelog-ext http://www.liquibase.org/xml/ns/dbchangelog/dbchangelog-ext.xsd">
+
+    <changeSet author="talend" id="add_sample_timeseries_data">
+        <!-- From anmn_ts_timeseries harvester -->
+        <sql>
+            CREATE TABLE ts_timeseries
```

Comment:
Considering this is test data, is it worth simplifying it a bit?  I think in general that test data should be the bare minimum needed to test a particular feature, otherwise it's difficult to ascertain what's *actually* having an effect on behaviour. Also, more to maintain as the code changes.

I know that's more work, so don't have to do it - just a comment.